src/internal/modules/charge-information/forms/use-abstraction-data.js

```diff
@@ -2,16 +2,37 @@
 
 const { get } = require('lodash');
 const Joi = require('@hapi/joi');
+const moment = require('moment');
 
 const { formFactory, fields } = require('shared/lib/forms/');
 const routing = require('../lib/routing');
 const { getActionUrl } = require('../lib/form-helpers');
 
+const getAddionalChoices = (chargeVersions) => {
+  const choices = [{ divider: 'or' }];
+  chargeVersions.map(cv => {
+    if (cv.status === 'current' || cv.status === 'superseded') {
```

Comment:
Not sure this check is necessary as the importer no longer imports draft charge versions - so all will be either current or superseded